import/dproto/intermediate.d

```diff
@@ -216,6 +236,10 @@ struct Field {
 	{
 		switch(fmt.spec) {
 			case 'p':
+				if(fmt.width == 3)
+				sink.formattedWrite("%s %s = %s%p; ",
```

Comment:
optional/required is no longer in proto3 -- it's in the change log:
https://github.com/google/protobuf/releases

That said, repeated is still in there, so it sounds like I have a bug.